packages/databricks-vscode/src/workspace-fs/WorkspaceFsAccessVerifier.ts

```diff
@@ -3,23 +3,77 @@ import {
     WorkspaceFsEntity,
     WorkspaceFsUtils,
 } from "@databricks/databricks-sdk";
-import {commands, Disposable, window} from "vscode";
+import {commands, Disposable, window, EventEmitter} from "vscode";
 import {ConnectionManager} from "../configuration/ConnectionManager";
 import {CodeSynchronizer} from "../sync";
 import {workspaceConfigs} from "../vscode-objs/WorkspaceConfigs";
+import {WorkspaceStateManager} from "../vscode-objs/WorkspaceState";
 
-async function switchToRepos() {
+export async function switchToRepos() {
     await workspaceConfigs.setSyncDestinationType("repo");
     commands.executeCommand("workbench.action.reloadWindow");
 }
 
+export async function switchToWorkspace() {
+    await workspaceConfigs.setSyncDestinationType("workspace");
+    commands.executeCommand("workbench.action.reloadWindow");
```

Comment:
Is it really required to reload the window? The experience would be better if we wouldn't have to.